payments-app-extension-redeemable/shopify.extension.toml.liquid

```diff
@@ -1,6 +1,6 @@
 # The version of APIs your extension will receive. Learn more:
 # https://shopify.dev/docs/api/usage/versioning
-api_version = "2023-07"
+api_version = "2024-04"
 
```

Comment:
perhaps add `checkout_payment_method_fields` in redeemables as well as it is also a part of the configuration: https://github.com/Shopify/shopify/blob/main/components/payment_processing/payments_partners/app/models/payments_partners/payments_app_extensions/redeemable_payments_app_extension.rb#L51

Reply:
Added 👍